apps/editing-toolkit/editing-toolkit-plugin/starter-page-templates/page-template-modal/index.js

```diff
@@ -309,6 +309,7 @@ class PageTemplateModal extends Component {
 			servicesTemplates: filter( this.props.templates, { category: 'services' } ),
 			teamTemplates: filter( this.props.templates, { category: 'team' } ),
 			homepageTemplates: sortBy( filter( this.props.templates, { category: 'home' } ), 'title' ),
+			comingSoonTemplates: filter( this.props.templates, { category: 'coming-soon' } ),
```

Comment:
Should wrap the feature flag around this? 

https://github.com/Automattic/wp-calypso/blob/master/apps/editing-toolkit/editing-toolkit-plugin/full-site-editing-plugin.php#L323

Reply:
Oh no, sorry. I wrote too soon.

It won't be in there since we're adding it in the coming soon plugin 🤦

Reply:
yea, it will only show if there are coming soon templates available, so it shouldn't affect anything when the feature flag is off

Reply:
Deleting transients (or not) between tests and switching the feature on off I was able to confirm this is safe (I think)